src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { push } from 'connected-react-router';
+
+import RenderInputTextField from '../RenderInputTextField';
+import RenderSelectField from '../RenderSelectField';
+import store from '../../data/store';
+import { AUDIT_TRACK, VERIFIED_TRACK, PROFESSIONAL_TRACK } from '../../data/constants';
+
+class CreateCourseForm extends React.Component {
+  getEnrollmentTrackOptions() {
+    return [
+      { label: 'Select enrollment track', value: '' },
+      { label: VERIFIED_TRACK.name, value: VERIFIED_TRACK.key },
+      { label: AUDIT_TRACK.name, value: AUDIT_TRACK.key },
+      { label: PROFESSIONAL_TRACK.name, value: PROFESSIONAL_TRACK.key },
+    ];
+  }
+
+  processOrganizations(organizations) {
+    const orgSelectList = [{ label: 'Select user organization', value: '' }];
+
+    if (organizations) {
+      organizations.forEach((org) => {
+        orgSelectList.push({ label: org.name, value: org.key });
+      });
+    }
+    return orgSelectList;
+  }
+
+  render() {
+    const {
+      handleSubmit,
+      organizations,
+      pristine,
+      submitting,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        <div className="create-course-form row">
+          <div className="col">
+            <form onSubmit={handleSubmit}>
+              <Field
+                name="courseOrg"
+                component={RenderSelectField}
+                options={this.processOrganizations(organizations)}
+                label={
+                  <React.Fragment>
+                    Organization
+                    <span className="required">*</span>
+                  </React.Fragment>
```

Comment:
Looking up what react.fragment does, I saw that you can also just write it like:

```
                  <>
                    Organization
                    <span className="required">*</span>
                  </>
```

But that feels... weird. And if I saw that, I wouldn't have known what to search for to see what it was.

Reply:
I agree that it looks like something I would delete.  The `<React.Fragment>` seems more clear to me.